espresso/environment/6_batch_inbox_test.go

```diff
@@ -0,0 +1,159 @@
+package environment_test
+
+import (
+	"context"
+	"math/big"
+	"testing"
+	"time"
+
+	env "github.com/ethereum-optimism/optimism/espresso/environment"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/geth"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/setuputils"
+	"github.com/ethereum-optimism/optimism/op-e2e/system/e2esys"
+	"github.com/ethereum-optimism/optimism/op-service/txmgr"
+	"github.com/ethereum-optimism/optimism/op-service/txmgr/metrics"
+	ethereum "github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/stretchr/testify/require"
+)
+
+const MOCK_AUTHENTICATOR_ADDR = "0xBAD8ae8e955D24D64D0b21b839ceF63075757Bc7"
+
+// TestE2eDevNetWithInvalidAttestation verifies that the batcher correctly fails to register
+// when provided with an invalid attestation. This test ensures that the batch inbox contract
+// properly validates attestations
+func TestE2eDevNetWithoutAuthenticatingBatches(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	launcher := new(env.EspressoDevNodeLauncherDocker)
+
+	system, _, err :=
+		launcher.StartDevNet(ctx, t, 0,
+			env.Config(func(cfg *e2esys.SystemConfig) {
+				cfg.L1Allocs[common.HexToAddress(MOCK_AUTHENTICATOR_ADDR)] = types.Account{
+					Code: hexutil.MustDecode("0x6080604052348015600e575f5ffd5b00fea26469706673582212202ea613f61d0ebf0addf2d722c3c06a3a613dd4932a28ed1c2e9aca8ca656808964736f6c634300081e0033"),
```

Comment:
What is this code? How is this hex string computed?

Reply:
I've just realized that if we have no contract deployed at the address we'll have the same exact result 🤦 
Removed this code.